control-plane/p4RuntimeSerializer.cpp

```diff
@@ -1317,6 +1383,22 @@ class P4RuntimeEntriesConverter {
             return stringRepr(k->to<IR::Constant>(), keyWidth);
         } else if (k->is<IR::BoolLiteral>()) {
             return stringRepr(k->to<IR::BoolLiteral>(), keyWidth);
+        } else if (k->is<IR::Member>()) {
```

Comment:
We already have a class to do this work, it is called SerEnumInstance.

Reply:
I changed the code to use SerEnumInstance.  It needs TypeMap, so I had to change more p4runtime serializer code.  

Incidentally, I checked in code changes last night, but this PR does not show any code segment commented upon as "Outdated".